spec/features/tasks_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+describe "viewing tasks" do
+  before(:each) do
+    o = Organisation.new
```

Comment:
Nice - we'll probably fork out to using FactoryGirl to avoid repeating these kind of declarations, because we'll use them in many tests.